lib/kernels/clink_kernels.cc

```diff
@@ -64,20 +64,20 @@ AsyncValueRef<double> SquareAdd(Argument<double> x, Argument<double> y,
 
 double Square(double x) { return x * x; }
 
-void OneHotEncoderLoad(Argument<std::string> path,
-                       Result<RCReference<OneHotEncoderModel>> result_model,
-                       KernelErrorHandler handler,
-                       const ExecutionContext &exec_ctx) {
-  auto model = OneHotEncoderModel::load(path.get(), exec_ctx.host());
+template <class T>
+void ModelLoad(Argument<std::string> path,
```

Comment:
Could we change this method to return a `Model` instead of a concrete subclass of `Model`?

Reply:
I'm not sure what is the question here. This method has been returning `Result<RCReference<Model>> result_model`, which has used `Model` instead of a subclass of `Model`.

Reply:
I see. That makes sense.